Missing MPI import in `cudecomp.h`

Following the removal of MPI, cutensor, and NCCL imports at the top of cudecomp.h in [this commit
](https://github.com/NVIDIA/cuDecomp/commit/df430e7158eaee8485830e404645e32d16f321cc) there is now an undefined reference to  `MPI_Comm` at this line of the include file:
https://github.com/NVIDIA/cuDecomp/blob/e0cf8713074739bbb057c058d0a386c217a73eeb/include/cudecomp.h#L186

This is a bit awkward because it forces the user to import manually MPI even in files that do not need to call the init function.  Could it be possible to add back that MPI import?

--- a/include/cudecomp.h
+++ b/include/cudecomp.h
@@ -40,6 +40,7 @@
 #include <string>
 
 #include <cuda_runtime.h>
+#include <mpi.h>
 
 #define CUDECOMP_MAJOR 0
 #define CUDECOMP_MINOR 2

